sample-swiftpm.xcodeproj/project.pbxproj

```diff
@@ -496,8 +499,9 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				BUNDLE_LOADER = "$(TEST_HOST)";
-				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 6653D4P657;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
NOTE: for some config you use `iPhone Developer`, while for other `Apple Development`